netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -249,51 +272,40 @@ public void inboundDataShouldForwardToStream() throws Exception {
     // Create a data frame and then trigger the handler to read it.
     // Need to retain to simulate what is done by the stream.
     ByteBuf frame = dataFrame(3, false).retain();
-    handler.channelRead(this.ctx, frame);
+    handler.channelRead(ctx, frame);
     verify(stream).transportDataReceived(eq(content), eq(false));
   }
 
   @Test
-  public void createShouldQueueStream() throws Exception {
-    // Disallow stream creation to force the stream to get added to the pending queue.
+  public void createShouldBufferStream() throws Exception {
```

Comment:
Does this test add anything? It seems this is already baked into several other tests.